### ISSUE TITLE
HA Italy

### ISSUE THREAD

**Issue by furetto72** [segalion/securitasdirect#7]
in the "alarm_control_panel.py" file
SECURITAS_STATUS = {
     '0': STATE_ALARM_DISARMED,
     'P': STATE_ALARM_ARMED_HOME,
     'Q': STATE_ALARM_ARMED_NIGHT,
     '1': STATE_ALARM_ARMED_AWAY
}
at least for Italy it appears to me:
SECURITAS_STATUS = {
     '0': STATE_ALARM_DISARMED,
     'Q': STATE_ALARM_ARMED_HOME,
     'C': STATE_ALARM_ARMED_NIGHT,
     'A': STATE_ALARM_ARMED_AWAY,
     '3': STATE_ALARM_ARMED_PERIM,
}

**Comment by segalion** [comment]
Thanks. I will add 'A' and 'C' states. 
Problem is 'Q' state that reports ARMED_NIGHT in spain and ARMED_HOME in Italy.
Please can you confirm?

**Comment by furetto72** [comment]
yes, I confirm 'Q': STATE_ALARM_ARMED_HOME, I add the perimeter:
 '3': STATE_ALARM_ARMED_PERI

**Comment by segalion** [comment]
Hi @furetto72 
Last version include state '3' as STATE_ALARM_ARMED_CUSTOM_BYPASS (the standar state in HA that we can asume as PERI)
Could you test it enabling this state in lovelace as [documentation](https://www.home-assistant.io/lovelace/alarm-panel/)

```
type: alarm-panel
entity: alarm_control_panel.securitas_xxxxxxx
name: Alarma Securitas
states:
  - arm_away
  - arm_home
  - arm_night
  - arm_custom_bypass
```

SECURITAS_STATUS = {
    STATE_ALARM_DISARMED: ['0',("1","32")],
    STATE_ALARM_ARMED_HOME: ['P',("311","202")],
    STATE_ALARM_ARMED_NIGHT: [('Q','C'),("46",)],
    STATE_ALARM_ARMED_AWAY: [('1','A'),("2","31")],
    STATE_ALARM_ARMED_CUSTOM_BYPASS: ['3',('???',)],
    STATE_ALARM_TRIGGERED: ['???',('13','24')],
}

**Comment by furetto72** [comment]
I'm sorry for my english. I tested the changes and found problems:
- if I try to disarm the panel it remains on I am disarming.
- STATE_ALARM_ARMED_NIGHT arms as STATE_ALARM_ARMED_HOME
- STATE_ALARM_ARMED_CUSTOM_BYPASS arms the perimeter but the panel does not display it and remains DISARMATED.
I changed your code like this:
SECURITAS_STATUS = {
     STATE_ALARM_DISARMED: ['0', ("1", "32")],
     STATE_ALARM_ARMED_HOME: ['Q', ("311", "202")],
     STATE_ALARM_ARMED_NIGHT: ['C', ("46",)],
     STATE_ALARM_ARMED_AWAY: ['A', ("2", "31")],
     STATE_ALARM_ARMED_CUSTOM_BYPASS: ['3', ('???',)],
     STATE_ALARM_TRIGGERED: ['???', ('13', '24')],
}
everything works except STATE_ALARM_ARMED_CUSTOM_BYPASS as mentioned above (disarmed state)

**Comment by furetto72** [comment]
log status PERI :
STATE_ALARM_ARMED_CUSTOM_BYPASS: ['3', ('**204**',)]

**Comment by segalion** [comment]
Thanks @furetto72 
Updated on code.

**Comment by Vartkat** [comment]
Hi,

Where are you looking to get these code ?
I'm in France and somtimes the status is not well reported.

**Comment by furetto72** [comment]
> Hi,
> 
> Where are you looking to get these code ?
> I'm in France and somtimes the status is not well reported.

https://github.com/Cebeerre/VerisureAPIClient - ACT -> LOG ACTIVITY

**Comment by meazzaflavio** [comment]
> log status PERI :
> STATE_ALARM_ARMED_CUSTOM_BYPASS: ['3', ('**204**',)]

Hello @furetto72 @segalion ,
I've modified the code as follow but I'm not able to activate perimetral:
SECURITAS_STATUS = {
    STATE_ALARM_DISARMED: ['0', ("1", "32")],
    STATE_ALARM_ARMED_HOME: ['P', ("311", "202")],
    STATE_ALARM_ARMED_NIGHT: [('Q', 'C'), ("46",)],
    STATE_ALARM_ARMED_AWAY: [('1', 'A'), ("2", "31")],
    STATE_ALARM_ARMED_CUSTOM_BYPASS: ['3', ('204',)],
    STATE_ALARM_TRIGGERED: ['???', ('13', '24')],
}

Do you know if there is something not correct in the code?
Thanks

**Comment by furetto72** [comment]
Ciao, io per ora ho abbandonato questo script in quanto ho riscontrato dei
seri problemi di loop nelle chiamate api al server di verisure, con
conseguente blocco dell'intero sistema di allarme.
Se vuoi ti invio gli ultimi codici che sono riuscito a recuperare.

Il dom 22 nov 2020, 13:10 meazzaflavio <notifications@github.com> ha
scritto:

**Comment by furetto72** [comment]
eccoli:
STATE_ALARM_DISARMED: ['0',("1","32")],
    STATE_ALARM_ARMED_HOME: ['P',("311","202", "203")],
    STATE_ALARM_ARMED_NIGHT: [('Q','C'),("46")],
    STATE_ALARM_ARMED_AWAY: [('1','A'),("2","31")],
    STATE_ALARM_ARMED_CUSTOM_BYPASS: [('3','C'),('204')],
    STATE_ALARM_TRIGGERED: ['???',('13','24')],

per ora io uso questo:
https://github.com/Cebeerre/VerisureAPIClient

Il giorno dom 22 nov 2020 alle ore 13:10 meazzaflavio <
notifications@github.com> ha scritto:
